main.py

```
@@ -39,19 +40,37 @@
+#         max_tokens=150,
+#         top_p=1.0,
+#         frequency_penalty=0.0,
+#         presence_penalty=0.0,
+#         stream=False,
+#     ),
+# )
+
+# input audio file from ./input.mp3
+data = akari.AkariData()
+dataset = akari.AkariDataSet()
+dataset.audio = akari.AkariDataSetType(main=open("input.mp3", "rb").read())
```

Ruff (0.8.2)

Use a context manager for opening files

(SIM115)